packages/es-components/src/components/containers/tabPanels/TabPanel.md

```diff
@@ -0,0 +1,30 @@
+Tab Panels allow us to store lots more data on the screen and only show the amount we want, when we want. This works by having a Tab contain context, or a Tab List that contains elements. Each direct child of the tab list must contain optiontext that will turn into the title in the dropdown. Tabs are able to have a full node inside instead of just text. This allows us to put in icons and other styling.
```

Comment:
Let's strip out references to `TabList` and how it was used until we get `TabList` back in here.

Reply:
To clarify, let's make this more relevant to how to use `TabPanel` and `TabPanel.Tab`. While the example is very straightforward it doesn't hurt to be explicit when we aren't directly displaying the props for `TabPanel.Tab`.